packages/beeq/src/components/alert/bq-alert.tsx

```diff
@@ -216,8 +219,12 @@ export class BqAlert {
   // ===================================
 
   render() {
+    const style = {
+      ...(this.border && { '--bq-alert--border-radius': `var(--bq-radius--${this.border})` }),
+    };
```

Comment:
Please leave an empty line between the const declaration and the return statement.